example/test/truffleToken.js

```diff
@@ -0,0 +1,66 @@
+contract('TruffleToken', function(accounts) {
+  it("should put 10000 TruffleTokens in the first account", function(done) {
+    var truf = TruffleToken.deployed();
```

Comment:
Same here. This is code coders will see. I want to remove any source of confusion.